app/services/search_service.rb

```diff
@@ -274,13 +274,17 @@ def create_nested_agg_body(body,top_key,field)
   def crowd_agg_facets(site:)
     params = self.params.deep_dup
     return {} if params.nil?
+  
 
     search_results = Study.search("*", aggs: [:front_matter_keys])
 
     aggs = search_results.aggs.to_h.deep_symbolize_keys
     keys = aggs[:front_matter_keys][:buckets]
       .map { |x| (x[:key]).to_s }
     facets = {}
+    if  self.params.dig(:crowd_buckets_wanted)
+      keys.select!{|key| self.params[:crowd_buckets_wanted]&.include?(key)}
```

Comment:
can you confirm where the list of crowd_buckets_wanted is being stored in the configuration of the workflow or is that outside the scope of this back end PR? Related, how does this behave if the request is desired to behave like the current behavior where ALL values are requested?

Reply:
Right now that if statement checks to see if crowd_buckets_wanted has a value. If it has a value of nil it does nothing. If its value contains something which means the front end sent in a value for it then it executes this code portion of selecting the keys specified in the array in crowd_bucket_wanted. This way was built with the assumption the front end sends in the array. If this needs to come from the configuration something needs to come from the front end telling the backend which pageview we are working off of. Right now the backend graphql request are unaware of the current page_view or site_view .

Reply:
@williamhoos Can you please verify that you're satisfied with the explanation above by marking the conversation as resolved or approving the pull request?

I would like to get this deployed to staging so we can test it out using GraphiQL until Rod finishes the front end piece (which should be ready either Wednesday or Thursday).